core/src/test/resources/splitFusionsData/data_fusions.txt

```diff
@@ -0,0 +1,3 @@
+Hugo_Symbol	Entrez_Gene_Id	Center	Tumor_Sample_Barcode	Fusion	DNA_support	RNA_support	Method	Frame	Comments
```

Comment:
Would you mind cleaning up the comments in this test data file and in the germline data file? It's probably best to strictly keep the data mocked so as to not contain any information resembling the original file, especially in the germline cases.